docs/section23/Rule23-9.md

```diff
@@ -40,16 +41,27 @@
+        - look at each terminal in the proposed zone: `for terminal_p in zone_p.terminals:`
+          - get the HVAC system associated with the terminal: `hvac_p = terminal_p.served_by_heating_ventilating_air_conditioning_system`
+          - get the fan system: `fan_system_p = hvac_p.fan_system`
+          - get the operation schedule hourly values: `operation_schedule_hourly_values_p = fan_system_p.operating_schedule.hourly_values`
+          - get the terminal minimum volume flowrate: `min_volume_p = terminal_p.minimum_airflow`
  
   **Rule Assertion:**  
   
-    - Case 1: the minimum volume flow rate is equal to the maximum of the min_ventilation_flowrate and 50% of the maximum_supply_flowrate - return PASS with a note: `if min_volume_flowrate == max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate): PASS; note = "The minimum volume flowrate is equal to the maximum of the minimum ventilation flowrate and 50% of the maximum supply flow rate.  If any airflow required to comply with codes or accredidation standards is MORE than this value, the minimum volume airflow should be set to this value.  We are not able to determine the airflow required to comply with codes or accreditation standards at this time, please double check that there are no additional codes or accreditation standards in regards to airflow." `  
-    
-    - Case 2: the minimum volume flow rate is greater than the maximum of the min_ventilation_flowrate and 50% of the maximum_supply_flowrate - return UNDETERMINED with a note: `elif min_volume_flowrate > max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate): UNDETERMINED; note = "The minimum volume flowrate is greater than the maximum of the minimum ventilation flowrate and 50% of the maximum supply flow rate.  This is correct IF the minimum volume flowrate is equal to any airflow required to comply with codes or accredidation standards, the system passes, otherwise it fails.  We are not able to determine the airflow required to comply with codes or accreditation standards at this time." `  
-    
-    - Case 3: all other cases, the rule fails: `else: FAIL`
+    - Case 1: If the proposed minimum volume flowrate (min_volume_p) is less than or equal tothe maximum of the other two values max(min_ventilation_flowrate and 0.5 * maximum_supply_flowrate), AND the minimum volume flowrate equals the maximum of (min_ventilation_flowrate and 0.5 * maximum_supply_flowrate) then we can provide a definite pass: `if(min_volume_p <= max(min_ventilation_flowrate and 0.5 * maximum_supply_flowrate) and min_volume_flowrate == max(min_ventilation_flowrate, 0.5 * maximum_supply_flowrate, min_volume_p): PASS`
```

Comment:
@KarenWGard I don't think this should pass if the min_volume_p equals the min_volume_flowrate unless they both equal 0.5 times maximum supply flow rate. I don't think it's definitive the way it's written.  I think it needs to be the following: if minimum_value_flow rate == 0.5 * maximum_supply_flow_rate == minimum_volume_p and minimum_value_flow rate >= min_ventilation_flowrate then pass

Please set me straight if I'm missing something or misunderstanding.